dask_expr/_expr.py

```diff
@@ -2603,6 +2603,56 @@ def _fusion_pass(expr):
     return expr
 
 
+class Fill(MapOverlap):
+    _parameters = [
+        "frame",
+        "how",
+        "axis",
+        "limit",
+    ]
+    _defaults = {"axis": 0, "limit": None}
+    before = 1
+    after = 0
+    enforce_metadata = True
+
+    @staticmethod
+    def func(frame, how, **kwargs):
```

Comment:
I think you can use M.ffill as operation instead of how?

Reply:
https://github.com/dask-contrib/dask-expr/pull/454/commits/9381794c1e7d4948d4be0365f9bb1d7143a0b886